docs/content/_navigation.json

```diff
@@ -1153,7 +1153,12 @@
           {
             "title": "Migrating from Airflow",
             "path": "/guides/migrations/migrating-airflow-to-dagster"
+          },
+          {
+            "title": "Coexistence of Airflow and Dagster",
```

Comment:
Would something like "Orchestrate your Airflow pipelines with Dagster" make sense?

I wonder if "Coexistence of Airflow and Dagster" might be confusing for users, mainly because of the term "coexistence". Could users think it's about using Dagster and Airflow in parallel? If so, the difference with the migration guide may not be obvious at first glance.

Reply:
+1 - I had a similar thought when I first looked at this.

Reply:
@maximearmstrong - I do believe one outcome of the Airlift project is for Dagster and Airflow to exist in parallel.

It will allow you to emit events from Airflow, and then build pipelines from those triggers in Dagster.

Another objective is that Dagster can act as a single pane of glass, and data catalog of your pipelines in Airflow and other orchestrators.

I think "Orchestrate your Airflow pipelines with Dagster" may not work in this case, as to me it implies that you can control Airflow from Dagster, whereas this tutorial is the inverse of that.

Reply:
That's fair. 

Is it possible to have several guides, one for each objective? Maybe we could have something like "Observe your Airflow pipelines with Dagster" and "Using Airflow events in Dagster"/"Trigger your Dagster pipelines from Airflow"?

If we want to limit ourselves to a single guide, that's fine too.

Reply:
That's a great idea - updated here: https://github.com/dagster-io/dagster/pull/22638/commits/e64c96996bd466c4aafd4145baf43df5ed75233d